config/services.yaml

```diff
@@ -19,3 +21,7 @@ services:
     App\Controller\:
         resource: '../src/Controller/'
         tags: [ "controller.service_arguments" ]
+
+    App\EventListener\CustomResponseHeaderSubscriber:
```

Comment:
Since autoconfigure is set to true, you don't need to to this manually since it is automatic.